src/main/kotlin/no/nav/familie/ef/personhendelse/inntekt/Inntektsendringer.kt

```diff
@@ -0,0 +1,54 @@
+package no.nav.familie.ef.personhendelse.inntekt
+
+import no.nav.familie.ef.personhendelse.client.OppgaveClient
+import no.nav.familie.ef.personhendelse.inntekt.vedtak.EfVedtakRepository
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import java.time.YearMonth
+
+@Component
+class Inntektsendringer(
+    val vedtakRepository: EfVedtakRepository,
+    val inntektClient: InntektClient,
+    val oppgaveClient: OppgaveClient
+) {
+
+    val secureLogger: Logger = LoggerFactory.getLogger("secureLogger")
+
+    fun beregnInntektsendringer() {
+        val personerMedVedtakList = vedtakRepository.hentAllePersonerMedVedtak()
```

Comment:
Hvis det finnes en revurdering på en behandling her så vill ikke det bli plukket opp, og vi kommer hente en person flere ganger. (dvs duplikat av en person i listen over vedtak)

Reply:
Sant. Endret spørringen til å ta høyde for dette, med den antakelsen om at en revurdering vil ha en høyere id enn førstegangsbehandlingen. Altså at den nyeste behandlingen alltid har høyest ID. Kunne også ha endret i lagringen til å kjøre update i stedet for insert for å ikke ha flere, men tenker det kan være fint med den historikken, og at det uansett må lages noe som rydder opp i utgåtte/fjernede vedtak.